src/test/java/unrc/dose/UserStatTest.java

```diff
@@ -24,21 +24,76 @@ public static void beforeAll() {
 		log.info("UserStatTest BeforeClass");
 		Base.open();
 	}
-	
-	@Before
-	public void beforeTest() {
-		Base.openTransaction();
-	}
 
 	@AfterClass
 	public static void afterAll() {
 		log.info("UserStatTest AfterClass");
 		Base.close();
 	}
-	
-	@After
-	public void afterTest() {
-		Base.rollbackTransaction();
+
```

Comment:
For tests it is recommended to use DBSpec. Furthermore, a beforeClass methods should be defined, to populate the DB with the information relevant to the tests.